www/templates/foundation/members/index.html

```diff
@@ -50,9 +50,17 @@ <h1 class="display-5 fw-bold mb-4">
                         <p>
```

Comment:
All of the text that was added in this page must be marked with translate or blocktranslate.